src/app/shared/footer/footer.component.ts

```diff
@@ -88,6 +89,27 @@ export class Footer {
       }, 1000)
     }
 
+
+    document.addEventListener("fullscreenchange", () => {
+      this.toggleFullscreen();
+    }, false);
+  
+    document.addEventListener("mozfullscreenchange", () => {
+      this.toggleFullscreen();
+    }, false);
+  
+    document.addEventListener("webkitfullscreenchange",() => {
+      this.toggleFullscreen();
+    }, false);
+  
+    document.addEventListener("msfullscreenchange", () => {
+      this.toggleFullscreen();
+    }, false);
+
+  }
+
+  private toggleFullscreen(): void {
+    this.isFullscreen = !this.isFullscreen
```

Comment:
I wonder if there is any way we could centralize this logic between the nav and footer. In Vue it seems like we could use mixins but I don't know if we have the same construct available in Angular.

Reply:
I'm thinking of putting it into a service and import the service in both components. I can try that after the meetings.